src/styles/main.scss

```diff
@@ -0,0 +1,70 @@
+@import '../styles/utils/variables';
+
+body {
+  margin: 0;
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+  justify-content: center;
+  height: 100vh;
+
+  font-family: Arial, sans-serif;
+  font-size: $font-size;
+}
+
+.calendar {
+  margin-left: 14px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+
+  width: calc(#{$day-size} * #{$week} + #{$padding} * 2);
```

Comment:
When using `calc` with SCSS variables, you need to make sure that the operations inside are correct. Here, you are using `#{$day-size} * #{$week}` which is fine, but for `#{$padding} * 2`, you should write it as `#{$padding} * 2` without the interpolation braces around the 2, because it's a standard number and not a variable.